azure-pipelines/compare-refdata.yml

```diff
@@ -95,13 +83,25 @@ jobs:
          sshPublicKey: $(opensupernova_pubkey)
          sshKeySecureFile: openSupernovaKey
 
+      - ${{ if eq(variables['Build.Reason'], 'IndividualCI') }}:
+        - bash: echo "##vso[task.setvariable variable=subfolder]releases"
+          displayName: "Set subfolder name"
+
+      - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+        - bash: echo "##vso[task.setvariable variable=subfolder]$(pr.number)"
+          displayName: "Set subfolder name"
+
+      - ${{ if eq(variables['Build.Reason'], 'Manual') }}:
+        - bash: echo "##vso[task.setvariable variable=subfolder]manual"
+          displayName: "Set subfolder name"
+
       - bash: |
-          ssh azuredevops@opensupernova.org "mkdir -p /home/azuredevops/public_html/files/refdata-results/$(pr.number)"
+          ssh azuredevops@opensupernova.org "mkdir -p /home/azuredevops/public_html/files/refdata-results/$(subfolder)"
```

Comment:
Put everything under a subfolder: PR (by pr number), manual triggered builds and release builds.